app/assets/stylesheets/vendor/main.css.erb

```diff
@@ -711,7 +711,7 @@ aside .module span { color: #000; }
     transition: 0.3s ease;
 }
 
-.btn:hover, .btn a:hover, .shareSave .btn a:hover, .menu-btn:hover, .toggle:hover {
+.btn:hover, .btn a:hover, .shareSave .btn a:hover, .toggle:hover {
```

Comment:
It might be good to keep the `.menu-btn:hover` to maintain consistency with the rest of the website.  If you're in a desktop browser and resize your window so its really skinny, you would have use of the `:hover` state on this button.  Unless there's another reason for getting rid of it?